sunpy/map/sources/hinode.py

```diff
@@ -21,6 +24,13 @@ class XRTMap(GenericMap):
 
     Hinode was launched on 22 September 2006 into a sun-synchronous orbit.
 
+    .. note::
+
+        If the required parameters for observator are not provided,
```

Comment:
TOOD: Check this renders nicely.

Reply:
You should be able to use
```
Notes
-----
```
To create a notes heading